runelite-client/src/main/java/net/runelite/client/plugins/camera/CameraPlugin.java

```diff
@@ -79,7 +79,10 @@
 	 * Whether or not the current menu has any non-ignored menu entries
 	 */
 	private boolean menuHasEntries;
-	
+
+	//Integer used to keep track of zoom value
+	private int currentZoom;
```

Comment:
Why are you doing this instead of just fetching the zoom value in `keyPressed`? This seems like this would not work if you zoomed with the mouse in combination with the keys.